src/Avalonia.Diagnostics/Diagnostics/DevToolsOptions.cs

```diff
@@ -22,5 +22,10 @@ public class DevToolsOptions
         /// Gets or sets the initial size of the DevTools window. The default value is 1280x720.
         /// </summary>
         public Size Size { get; set; } = new Size(1280, 720);
+
+        /// <summary>
+        /// Get or set the startup screen number where the DevTools window will be displayed.
+        /// </summary>
+        public int? StartupScreen { get; set; }
```

Comment:
Should it reference Screen object instead? We will have problems with disconnected monitors in either way.
Or if we leave "int?" prop, it should be StartupScreenIndex

Reply:
I check here https://github.com/AvaloniaUI/Avalonia/blob/02fd0006a9ddd11f88e3e7a821d779aa46c35e0d/src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs#L35 if monitor  index exist.